src/transformers/integrations/deepspeed.py

```diff
@@ -39,9 +39,6 @@ def is_deepspeed_available():
     # AND checking it has an author field in the metadata that is HuggingFace.
     if package_exists:
         try:
-            if is_torch_mlu_available():
-                _ = importlib_metadata.metadata("deepspeed-mlu")
-                return True
```

Comment:
This is indeed not required anymore with the latest version of deepspeed but it will be breaking for users that doesn't have it yet ! If you don't want to rely on deepspeed-mlu anymore, you should inform the users that they need to install the latest deepspeed